doc/devices/ibmq.rst

```diff
@@ -1,84 +0,0 @@
-IBM Q Experience
-================
-
-PennyLane-Qiskit supports running PennyLane on IBM Q hardware via the ``qistkit.ibmq`` device.
-You can choose between different backends - either simulators tailor-made to emulate the real hardware,
-or the real hardware itself.
-
-Accounts and Tokens
```

Comment:
Need to write new docs explaining to use ibm_runtime_service instead. Separate PR!